src/locales/en-US/settingDrawer.ts

```diff
@@ -15,6 +15,11 @@ export default {
   'app.setting.themecolor.geekblue': 'Geek Glue',
   'app.setting.themecolor.purple': 'Golden Purple',
   'app.setting.navigationmode': 'Navigation Mode',
+  'app.setting.regionalsettings': 'Regional Settings',
```

Comment:
Available in Chinese translation, but not in the US translation: https://github.com/ant-design/ant-design-pro-layout/blob/master/src/locales/zh-CN/settingDrawer.ts

This looks to be causing this issue in the preview on https://preview.pro.ant.design/dashboard/analysis:

<img width="298" alt="Screen Shot 2020-07-06 at 7 19 43 PM" src="https://user-images.githubusercontent.com/463175/86665555-be42e280-bfbd-11ea-9d8b-04eac30fb6c3.png">